tests/src/Functional/SettingsTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\collabora_online\Functional;
+
+use Drupal\Core\Url;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests the Collabora configuration.
+ */
+class SettingsTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'collabora_online',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * Tests the configuration for the Collabora settings form.
+   */
+  public function testSettingsForm(): void {
+    $assert_session = $this->assertSession();
+
+    // User without permission can't access the configuration page.
+    $this->drupalGet(Url::fromRoute('collabora-online.settings'));
+    $assert_session->pageTextContains('You are not authorized to access this page.');
+    $assert_session->statusCodeEquals(403);
+
+    // User with permission can access the configuration page.
+    $user = $this->createUser(['administer site configuration']);
+    $this->drupalLogin($user);
+    $this->drupalGet(Url::fromRoute('collabora-online.settings'));
+    $assert_session->statusCodeEquals(200);
+
+    // Check fields and default values.
+    $this->drupalGet(Url::fromRoute('collabora-online.settings'));
+    $assert_session->fieldValueEquals('Collabora Online server URL', 'https://localhost:9980/');
+    $assert_session->fieldValueEquals('WOPI host URL', 'https://localhost/');
+    $assert_session->fieldValueEquals('JWT private key ID', 'cool');
+    $assert_session->fieldValueEquals('Access Token Expiration (in seconds)', '86400');
+    $assert_session->fieldValueEquals('Disable TLS certificate check for COOL.', '');
+    $assert_session->fieldValueEquals('Allow COOL to use fullscreen mode.', '1');
+
+    // Check that fields values are saved.
```

Comment:
```suggestion
    // Change the form values, then submit the form.
```

Reply:
In general these comments could be bikeshed forever, my suggestions may not be ideal either.

Imo "Check xyz" is typically too vague, I prefer to say "Do x" and then a separate comment further down "Verify that y happens". Or if we have one comment for both parts, we could just say "Test that x causes y".

Reply:
We could even be more simple, and instead of "Verify that all form fields appear" we just write "All form fields appear, filled with default values".

Reply:
Its ok for me, as long is clear what the code is doing, most of the times I use "check" for tests but "verify" it's ok too.